.github/rpm/gpg_pub.asc

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
public GPG key, will need to update once we generate the private key for twilio-cli